static/app/utils/routeAnalytics/useDisableRouteAnalytics.spec.jsx

```diff
@@ -0,0 +1,22 @@
+import {render} from 'sentry-test/reactTestingLibrary';
```

Comment:
Is it possible to make the the `*.spec.jsx` files to be in typescript? 👀

Reply:
Yes but is that something we agreed to as an organization? I’ve personally seen no real value in Typescript tests.

Reply:
The point is to test the particular spec. Why test the TS part of it? It should just work because the code compiles.

Reply:
@scefali usually it's fair game that new tests be in typescript. Pre-existing tests in `.spec.jsx` doesn't need to be converted.